src/main/resources/hudson/plugins/mercurial/MercurialChangeSetList/digest.jelly

```diff
@@ -3,6 +3,7 @@
   when a build history link (or number) is followed
   e.g http://<hudson server>/job/<project>/<build number>/
 -->
+<?jelly escape-by-default='true'?>
```

Comment:
Digged into the code for a bit. 
Jesse is right, the <code>msgAnnotated</code> returns HTML: https://github.com/jenkinsci/jenkins/blob/master/core/src/main/java/hudson/scm/ChangeLogSet.java#L248-260